### ISSUE TITLE
Automatically turn off tabs when screen reader detected

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#95682]
Talking to our screen reader users most of them disable tabs since it makes navigation a lot easier.
Also imho tabs are a very visual thing so it might make sense to automatically disable them when screen reader is attached. For example we could change what is the default of the setting `workbench.editor.showTabs`, so users could still overwrite it if they want to

Just creating this issue for discussion.

**Comment by alex19EP** [comment]
can you explain which tabs in question?

**Comment by isidorn** [comment]
@alex19EP yes, sorry for not being very clear.
Once a file is opened in the editor area we open it in an editor tab. Same as how Chrome and other browsers open webpages in tabs, we do the same in VS Code.

The only benefit that screen reader users have from that is that they can be tabbed (using the tab key on the keybaord) into.

Another way to put it is if you enable and disable the following setting you can disable and enable them, and you can check if there is a difference for your flow `workbench.editor.showTabs`

**Comment by alex19EP** [comment]
yep. i checked not much difference for me.
so i am +1

**Comment by MarcoZehe** [comment]
Well, I turned that setting off right at the start when using VS Code, since @Zersiax and I believe @DerekRiemer recommended it. So I would not be opposed to VS Code disabling those tabs by default.

**Comment by webczat** [comment]
I am generally unsure about the policy of changing settings because accessibility is enabled, with some exceptions probably. If something makes not much difference one can as well ignore it. I actually haven't touched any tab related setting and I am still wondering what is it all about, so... Does this give any benefits as opposed to just hiding something that we don't need but that does not make much difference? And does enabling of tabs actually charm accessibility?

**Comment by alex19EP** [comment]
@webczat i think it's saves time if you move with tab near the editor or use revue mode.

**Comment by webczat** [comment]
well for me it/s all the same, would prefer shortcuts like ctrl+b to just jump to open editor. one tab more, one tab less... I would say this should be something left to discretion of the user.
Note I am likely one of the least strict people when it goes to accessibility. In circumstances some people would find something inaccessible, for me it's enough that I can use something, and then if I somehow find an accessibility bug I often do not find it a high priority when it goes to reporting. So that's partially why I don't care so much about such stuff.

**Comment by jvesouza** [comment]
How wonderful, I didn't know it was possible to disable this option.
I disabled it here and I think navigation is better.
To navigate between the various open editors we can use ctrl + pageDown and ctrl + pageUp.

**Comment by jvesouza** [comment]
It seems that when we disable the option workbench.editor.showTabs, orca stops announcing the file name when using ctrl + pageUp / Down or alt + n.
To reproduce you need to include the following line in the settings.json:
"workbench.editor.showTabs": true,

Would I be better off reporting this in another issue?

**Comment by jvesouza** [comment]
Please disregard my previous comment. @joanmarie Already fixed the problem in orca, even without knowing it!
Shortly after I made the comment, some updates were made to orca and these updates seem to have fixed the problem.

**Comment by isidorn** [comment]
Thanks all for feedback.
How the things hold we will probably disable tabs automatically when screen reader is detected start of next Milestone in may, so we get a whole month to gather feedback.

**Comment by isidorn** [comment]
After thinking more about this I have decided to not automatically disable tables after screen reader is detected.
Thus closing as wont-fix
